src/geos_funcs_O_b.c

```diff
@@ -0,0 +1,288 @@
+#define PY_SSIZE_T_CLEAN
+
+#include <Python.h>
+#include <math.h>
+
+#define NO_IMPORT_ARRAY
+#define NO_IMPORT_UFUNC
+#define PY_ARRAY_UNIQUE_SYMBOL shapely_ARRAY_API
+#define PY_UFUNC_UNIQUE_SYMBOL shapely_UFUNC_API
+#include <numpy/ndarraytypes.h>
+#include <numpy/npy_3kcompat.h>
+#include <numpy/ufuncobject.h>
+
+#include "fast_loop_macros.h"
+#include "geos.h"
+#include "pygeos.h"
+#include "pygeom.h"
+#include "signal_checks.h"
+
+/* ========================================================================
+ * WRAPPER FUNCTIONS
+ * ========================================================================
+ *
+ * Function signature for operations that take a Python object and return a bool: O->b.
+ * These functions do not raise errors on non-geometry objects (unlike Y_b functions).
+ *
+ * Parameters:
+ *   obj: Input Python object (may be a geometry or any other type)
+ *
+ * Returns:
+ *   0 for false, 1 for true
```

Comment:
Not entirely correct? It returns an error code, and return false/true to the return pointer (the GEOS functions it wraps do that, though)

Reply:
I suppose the difference here is that there is no "core operation logic" section that typically used this pattern?